src/transformers/onnx/convert.py

```diff
@@ -78,21 +82,83 @@ def export(
     Returns:
 
     """
-    if not is_torch_available():
-        raise ImportError("Cannot convert because PyTorch is not installed. Please install torch first.")
+    from ..file_utils import torch_version
 
-    import torch
-    from torch.onnx import export
+    if not (is_torch_available() or is_tf_available()):
+        raise ImportError(
+            "Cannot convert because neither PyTorch nor Tensorflow are not installed. Please install torch or tensorflow first."
+        )
 
-    from ..file_utils import torch_version
+    if is_torch_available():
```

Comment:
Now that I've had a chance to look at the full code, I'm thinking it might be a good idea to refactor the PyTorch / TensorFlow specific exports into two functions:

- `export_pytorch()`
- `export_tensorflow()`

Then we can call these functions within the main `export()` function which will make the code a lot easier to read and test. Let me know if you'd like to tackle that or I can do it myself if you prefer :)

Reply:
Yes ok, done thank you :)